examples/stablecoin.py

```
@@ -0,0 +1,41 @@
+# Stablecoin (ERC-20 style) Smart Contract Example
+#
+# This contract implements a minimal fungible token.
+#
+# Valid Payloads:
+# - 'mint:<amount>'
+# - 'transfer:<recipient_address>:<amount>'
+
+if msg['data'].startswith('mint:'):
```

Ruff (0.15.15)

[error] Undefined name `msg`

(F821)